koans/ExceptionsKoans.php

```diff
@@ -12,10 +12,13 @@
 
 class ExceptionsKoans extends TestCase
 {
+    // Resources for learning about Exceptions => https://www.w3schools.com/php/php_exceptions.asp
+
     /**
+     * @test
      * @testdox Exceptions can be thrown using the `throw` keyword
      */
-    public function testThrowException()
+    public function useExpectExceptionMessageToMakeSureTheExceptionMessageIsWhatYouExpect()
```

Comment:
It seems to me that you are giving more information than necessary, couldn't it really be expectExceptionMessage?